trail/upload/process_uploads/processors.py

```diff
@@ -0,0 +1,286 @@
+
+        unitSphereCenter = np.array([
+            np.cos(centerDec) * np.cos(centerRa),
+            np.cos(centerDec) * np.sin(centerRa),
+            np.sin(centerDec)
+        ])
+
+        unitSphereCorner = np.array([
+            np.cos(cornerDec) * np.cos(cornerRa),
+            np.cos(cornerDec) * np.sin(cornerRa),
+            np.sin(cornerDec)
+        ])
+
+        unitRadius = np.linalg.norm(unitSphereCenter-unitSphereCorner)
+
+        # do we really need this?
+        #standardizedWcs["ctype"] = wcs.wcs.ctype
+        #standardizedWcs["crpix1"] = wcs.wcs.crpix[0]
+        #standardizedWcs["crpix2"] = wcs.wcs.crpix[1]
+
+        standardizedWcs["radius"] = unitRadius
+
+        standardizedWcs["center_x"] = unitSphereCenter[0]
+        standardizedWcs["center_y"] = unitSphereCenter[1]
+        standardizedWcs["center_z"] = unitSphereCenter[2]
+                                 
+        standardizedWcs["corner_x"] = unitSphereCorner[0]
+        standardizedWcs["corner_y"] = unitSphereCorner[1]
+        standardizedWcs["corner_z"] = unitSphereCorner[2]
+
+        return standardizedWcs
+
+    def standardized_header_metadata(self):
+        """Normalizes FITS header information of the uploaded file and returns
+        a dictionary with standardized, as understood by trailblazer, header
+        keys.
+
+        Rubin's ``astro_metadata_translator`` is used to perform the
+        translation for the most popular instruments. 
+
+        Returned dict contains the following keys:
+        * obs_[lon, lat, height] - Observatory coordinates
+        * name - name of translator class that performed the translation, oten
+                 a simple(r) instrument name
+        * telescope
+        * instrument
+        * science_program
+        * datetime_[begin, end] - start and end of exposure
+
+        Returns
+        -------
+        standardizedKeys : `dict`
+          A dictionary with standardized header keys and values.
+        """
+        standardizedKeys = {}
+
+        # see about edgecases later, as is it seems this always
+        # reads the primary header
+        translator = translate.MetadataTranslator.determine_translator(self.header,
+                                                                       filename=self._upload.filename)
+
+        if translator.can_translate(self.header, filename=self._upload.filename):
+            translated = translator(self.header, filename=self._upload.filename)
+
+            location = translated.to_location()
+
+            standardizedKeys["obs_lon"] = location.lon.value
+            standardizedKeys["obs_lat"] = location.lat.value
+            standardizedKeys["obs_height"] = location.height.value
+
+            # translator names are easy everyday names
+            # the rest I don't know how they'll vary over different
+            # instruments but I can assume potentially by a lot?
+            standardizedKeys["name"] = translated.name
```

Comment:
Call this `metadata_translator_name` or something a bit more specific so people don't misconstrue it as, e.g., the name of a satellite or other target in the image